deploy/helm-chart/kubernetes-replicator/templates/rbac.yaml

```diff
@@ -3,6 +3,7 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: {{ include "kubernetes-replicator.serviceAccountName" . }}
+  automountServiceAccountToken: {{ include kubernetes-replicator.serviceAccount.automountServiceAccountToken" . }}
```

Comment:
According to the [docs](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#use-the-default-service-account-to-access-the-api-server), `.automountServiceAccountToken` is a root-level field, and not a metadata property.